terraform/modules/emblem-app/main.tf

```diff
@@ -78,3 +78,102 @@ resource "google_storage_bucket_iam_member" "sessions-iam" {
   role   = "roles/storage.objectAdmin"
   member = "serviceAccount:${google_service_account.website_manager.email}"
 }
+
+############################
+# E2E Delivery testing IAM #
+############################
+
+resource "google_project_iam_member" "delivery_gae_viewer_iam_member" {
```

Comment:
future/nit: Per https://cloud.google.com/docs/terraform/best-practices-for-terraform#naming-convention, we should avoid including the resource type (or part of it) in the resource name. Flagging this for awareness that in the future this may be a requirement. (Team discussion on how to adopt and apply these best practices pending)

Reply:
I went ahead and removed the `_iam_member` suffix.

(Naturally, other occurrences of this are outside the scope of this PR. 😝 )